### PR TITLE
Adding Unity 2017.2 and 2017.3 support, as well as additional UWP support

### PR DESCRIPTION
**Pull request by keveleigh**
This PR adds a few newly generated folders for both UWP and Unity 2017 to the gitignore.

It also add versioning `#if` statements where APIs have changed between Unity versions.

### CODE REVIEW

**Comment by MSblgross** [comment]
Thanks for the contribution. I should be committing a change soon that does some of the work to upgrade. As a note, you don't want the IntegrationTest utilities in a version past 5.6 since Unity changed how integration tests are run,

File: UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs [hunk 1]
```diff
@@ -774,7 +781,11 @@ protected virtual IEnumerator LoadBuffer(string sourceUri, GLTF.Schema.Buffer bu
 				{
 					var www = UnityWebRequest.Get(Path.Combine(sourceUri, uri));
 
+#if UNITY_2017_2_OR_NEWER
```
Comment (StephenHodgson) [inline UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs]: @keveleigh pretty sure this was introduced in 2017.1, but I may be wrong.
Please double check.

**Comment by StephenHodgson** [comment]
@keveleigh there wasn't any shader changes required for 2017.3?

File: UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs (hunk 1 above)
Comment (keveleigh) [reply UnityGLTF/Assets/UnityGLTF/Scripts/GLTFSceneImporter.cs]: Double checked, and 2017.1 does not have `SendWebRequest`

image

**Comment by blgrossMS** [comment]
I merge these changes in manually to master. Sorry it took so long, thanks for the contribution!

**Pull request closed without merging**